src/main/pages/che-7/installation-guide/proc_adding-custom-certificates-to-trust-store.adoc

```diff
@@ -0,0 +1,34 @@
+[id="adding-custom-certificates-to-trust-store_{context}"]
+= Adding custom public SSL certificates to {prod-short} trust-store
+
+This procedure describes how to configure {prod-short} to be able to handle HTTP requests from untrusted sources.
+
+.Procedure
+. Save the certificate that you want to apply to `ca.crt` file.
+. Create a new *configMap* with the certificate:
```

Comment:
configMap or secret?

Reply:
ConfigMap, I don't see any reasons to use secret